modules/apps/commerce/commerce-wish-list-service/service.xml

```diff
@@ -112,6 +112,7 @@
 		</finder>
 	</entity>
 	<exceptions>
+		<exception>CommerceWishListCannotDelete</exception>
```

Comment:
My PR was rejected I'm waiting for an answer on which pattern to use for this exception..